be/src/exprs/agg/any_value.h

```diff
@@ -26,62 +26,39 @@
 
 namespace starrocks {
 
-template <LogicalType LT>
-struct AnyValueAggregateData {
-    using T = AggDataValueType<LT>;
-
-    T result;
-    bool has_value = false;
-
-    void reset() {
-        result = T{};
-        has_value = false;
-    }
-};
-
-template <LogicalType LT, typename State>
-struct AnyValueElement {
-    using RefType = AggDataRefType<LT>;
-
-    void operator()(State& state, RefType right) const {
-        if (UNLIKELY(!state.has_value)) {
-            AggDataTypeTraits<LT>::assign_value(state.result, right);
-            state.has_value = true;
+struct AnyValueState {
+    void update(FunctionContext* ctx, const Column& column, size_t offset) {
+        if (!has_fill) {
+            data_column = ctx->create_column(*ctx->get_arg_type(0), false);
+            data_column->append(column, offset, 1);
```

Comment:
does this reject null?

Reply:
why the column is not nullable?

Reply:
no, don't reject null

Reply:
`data_column = ctx->create_column(*ctx->get_arg_type(0), false);` set nullable as `true` instead of `false`?

Reply:
see related code, any_value actually ignores null, its  `process_null` is empty from `NullableAggregateFunctionUnary`

Reply:
> `data_column = ctx->create_column(*ctx->get_arg_type(0), false);` set nullable as `true` instead of `false`?

I packaged NestNullAggregateFunction, so `ANY_VALUE` only need handle none-null